### DISCUSSION TITLE
usage of 'yarn lint' script ?

### DISCUSSION THREAD

**Discussion by underwater** [ghostfolio/ghostfolio#2772] [General]
before a p/r was trying to perform linting on the project using
`yarn lint`

However, I get
`
 NX   Unable to resolve @nrwl/linter:eslint.
 Cannot find module '@nrwl/linter/package.json'
`

searching on the net, suggested that the the @nrwl/linter module is not found in my project's node_modules directory.  
Did anyone else encouter this ?

**Comment by dtslvr** [comment]
This is probably related to https://github.com/ghostfolio/ghostfolio/pull/2767.

**Reply by underwater** [reply]
tried it and got this error
`
Linting "api"...

 >  NX   ESLint configuration in apps/api/.eslintrc.json » ../../.eslintrc.json is invalid:

        - Property "extends" is the wrong type (expected string but got `[null,"plugin:storybook/recommended"]`).
        - Property "extends[0]" is the wrong type (expected string but got `null`).
        - "extends" should match exactly one schema in oneOf. Value: [null,"plugin:storybook/recommended"].
   
   Referenced from: /home/fares/repo/_projects/ghostfolio/apps/api/.eslintrc.json
   Pass --verbose to see the stacktrace.


 >  NX   Ran target lint for project api (613ms)
 
    ✖    1/1 failed
    ✔    0/1 succeeded [0 read from cache]
 
   View structured, searchable error logs at https://nx.app/runs/SDeoEU7RfZ

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
`
Not sure yet, but will let u know if I find a solution.

**Reply by dtslvr** [reply]
I have created an issue: https://github.com/ghostfolio/ghostfolio/issues/2782
